stylesheets/components/toast.css

```diff
@@ -0,0 +1,30 @@
+@import url(/stylesheets/main.css);
+
+.toast-msg {
+     padding: 0.5rem 1rem;
+     background-color: var(--white);
+     border: 0.1rem solid var(--black);
+     font-size: larger;
```

Comment:
Try to have a variable or give value instead of directly specifying larger

Reply:
ok done